src/core/server/services/users/index.ts

```diff
@@ -585,6 +587,11 @@ export async function ignore(
     throw new UserNotFoundError(userID);
   }
 
+  const userToBeIgnoredIsStaff = await userIsStaff(targetUser);
```

Comment:
Assuming we can remove `async`, remove the `await`, and probably just put it into the `if (...) {` clause as the value isn't used outside this check.

Reply:
You're right, async removed as well as the await.